client/app/dedicated/server/interfaces/interfaces.controller.js

```diff
@@ -0,0 +1,2 @@
+export default class DedicatedServerInterfacesCtrl {
```

Comment:
Well ?

Reply:
Yes, we could remove these great controller Class :)